### ISSUE TITLE
incorrect note in aesks1i.h

### ISSUE THREAD

**Issue by dingiso** [riscv-software-src/riscv-isa-sim#730]
line 23 the note rotate left shoud be rotate right
**/riscv/insns/aesks1i.h**
```c
if(enc_rcon != 0xA) {
    temp    = (temp >> 8) | (temp << 24); // Rotate left by 8
    rcon    = round_consts[enc_rcon];
}
```

**Comment by aswaterman** [comment]
@ben-marshall Just confirming the bug is in the comment, not the code.

Can one of you submit a PR with the fix?

**Comment by ben-marshall** [comment]
Ah shucks yes, it should indeed say "rotate right". See [section 3.10, page 19](https://github.com/riscv/riscv-crypto/releases/download/v0.9.2-scalar/riscv-crypto-spec-scalar-v0.9.2.pdf).

I'll put in a PR unless @dingiso gets there first.

**Comment by aswaterman** [comment]
Resolved by #733
